lms/envs/common.py

```diff
@@ -2742,4 +2742,4 @@
 # This is an arbitrary hard limit.
 # The reason we introcuced this number is because we do not want the CCX
 # to compete with the MOOC.
-CCX_MAX_STUDENTS_ALLOWED = 200
+CCX_MAX_STUDENTS_ALLOWED = 200
```

Comment:
Same here. I suspect your text editor is incorrectly configured and is stripping the trailing newlines.